my-vue3-app/src/components/ReportList.vue

```diff
@@ -51,6 +58,19 @@ export default defineComponent({
     async onPage({ page, rows: pageSize }: {page: number; rows: number}) {
       this.fetchReports({ page, pageSize });
```

Comment:
`this.fetchReports({ page, pageSize });` -> 
```ts
this.fetchReports({ page, pageSize, filters: { nick: this.nick } });
```

Reply:
even better create some new method `fetchWith` which will be getting only changes and use previous call arguments
so here you can just call
```ts
this.fetchWith({ page, pageSize}); // actually call with { page, pageSize, filters: { nick: latCalledNick } }
```

Reply:
About the first msg. You mean from now on function fetchReports should receive 3 arguments ?

Reply:
No. 2